cl_dll/forcemodel.cpp

```diff
@@ -479,6 +479,38 @@ namespace force_model
 		const auto local_player_index = gEngfuncs.GetLocalPlayer()->index;
 		if (!strcmp(g_PlayerExtraInfo[local_player_index].teamname, g_PlayerExtraInfo[player_index + 1].teamname))
 			teammate_enemy_model_overrides_cache[player_index] = teammate_model_override;
+		else if(g_IsSpectator[local_player_index])
```

Comment:
Maybe put this if separately right before? So it's like
```cpp
const auto local_player_index = gEngfuncs.GetLocalPlayer()->index;

if (g_IsSpectator[local_player_index])
{
    // ...
    return;
}

if (!strcmp(g_PlayerExtraInfo[local_player_index].teamname, g_PlayerExtraInfo[player_index + 1].teamname))
```

Reply:
Done &